metadata-ingestion/src/datahub/ingestion/source/schema/json_schema.py

```diff
@@ -280,18 +290,29 @@ def _load_one_file(
                 entityUrn=dataset_urn, aspect=models.StatusClass(removed=False)
             ).as_workunit()
 
-            yield MetadataChangeProposalWrapper(
-                entityUrn=dataset_urn,
-                aspect=models.DatasetPropertiesClass(
-                    externalUrl=JsonSchemaTranslator._get_id_from_any_schema(
-                        schema_dict
-                    ),
-                    name=dataset_simple_name,
-                    description=JsonSchemaTranslator._get_description_from_any_schema(
-                        schema_dict
-                    ),
-                ),
-            ).as_workunit()
+            dataset_patch_builder = DatasetPatchBuilder(dataset_urn).set_display_name(
+                dataset_simple_name
+            )
+            try:
+                external_url = JsonSchemaTranslator._get_id_from_any_schema(schema_dict)
+                external_url_parsed = urlparse(external_url)
+                if not all([external_url_parsed.scheme, external_url_parsed.netloc]):
+                    external_url = None
+                else:
+                    dataset_patch_builder.add_custom_property(
+                        "externalUrl", external_url
+                    )
+            except:
+                # Nothing to do in this case
+                pass
+            description = JsonSchemaTranslator._get_description_from_any_schema(
+                schema_dict
+            )
+            if description:
+                dataset_patch_builder.add_custom_property("description", description)
```

Comment:
this is for custom properties in the properties tab - use `set_description` instead